pkg/loader/loader.go

```diff
@@ -318,14 +325,14 @@ func SplitToolRef(targetToolName string) (toolName, subTool string) {
 }
 
 func isOpenAPI(data []byte) bool {
-	var version struct {
-		OpenAPI string `json:"openapi" yaml:"openapi"`
+	var fragment struct {
+		Paths map[string]any `json:"paths,omitempty"`
```

Comment:
We want to check for the existence of the `paths` map, rather than the version string, since the version string is sometimes missing.